src/conf_mode/protocols_mpls.py

```diff
@@ -68,7 +70,8 @@ def generate(mpls):
         mpls['new_frr_config'] = ''
         return None
 
-    mpls['new_frr_config'] = render_to_string('frr/ldpd.frr.tmpl', mpls)
+    mpls['new_frr_config'] = render_to_string('frr/ldpd.frr.tmpl', mpls,
+                                              trim_blocks=True)
```

Comment:
Trim blocks is enabled by default and that signature is now invalid.

Reply:
Ahhh ok that I did not know. Removing.

Thank you sir